### ISSUE TITLE
closing brackets in a line

### ISSUE THREAD

**Issue by FlorianZipser** [korpling/pepperModules-PTBModules#2]
The current implementation does not support single closing brackets in a line. For instance the TueBaDS corpus uses a dialect of ptb, in which lines can contain only closing brackets. For instance:

%% sent. no. 1
(
  (DM
    (NX
      (ADJX
        (ADJA:HD guten)
      )
      (NN:HD Tag)
    )
  )
  ($, ,)
  (DM
    (NX
      (NN Frau)
      (NE Weiss)
    )
  )
  ($, ,)
  (SIMPX
    (VF
      (NX:ON
        (PPER:HD wir)
      )
    )
    (LK
      (VXFIN:HD
        (VMFIN:HD m"ussen)
      )
    )
  ($, ,)
    (SIMPX:PAR
      (LK
        (VXFIN:HD
          (VVFIN:HD glaube)
        )
      )
      (MF
        (NX:ON
          (PPER:HD ich)
        )
      )
    )
  ($, ,)
    (MF
      (ADVX:MOD
        (ADV:HD noch)
      )
      (NX:OA
        (NX:HD
          (ART eine)
          (NN:HD Gesch"aftsreise)
        )
        (PX
          (APPR nach)
          (NX:HD
            (NE:HD Hannover)
          )
        )
      )
    )
    (VC
      (VXINF:OV
        (VVINF:HD machen)
      )
    )
  ($, ,)
    (NF
      (SIMPX:MOD
        (C
          (KOUI um)
        )
        (MF
          (NX:OA
            (PPOSAT unsere)
            (NN:HD Filiale)
          )
          (ADVX:V-MOD
            (ADV:HD dort)
          )
        )
        (VC
          (VXINF:HD
            (PTKZU:HD zu)
            (VVINF besuchen)
          )
        )
      )
    )
  )
  ($. .)
)

This seems to be caused by line 130 -132 in the PTBImporterMapper:

```
        while (!(strValidate.trim().startsWith("("))) {
            strValidate = br.readLine();
        }
```

but expanding that for closing brackets causes test failures. It seems that the following regexs can not deal with that.

**Comment by amir-zeldes** [comment]
OK, this should work now – lines still have to begin with some sort of bracket (cf. the lines with starting with %% below, which should be ignored).

**Comment by amir-zeldes** [comment]
Fixed in f43843f6f38eae78aae9380425de0f5cf488cb52
